index.d.ts

```diff
@@ -60,6 +60,13 @@ export interface FastifySwaggerOptions {
    */
   staticCSP?: boolean | string | Record<string, string | string[]>
   transformStaticCSP?: (header: string) => string
+  /**
+   * route hooks
+   */
+  uiHooks: {
```

Comment:
It should be optional.
```suggestion
  uiHooks?: {
```